tests/Behat/Context/Reorder/ReorderContext.php

```diff
@@ -124,23 +142,23 @@ public function iShouldBeNotifiedThatPromotionIsNoLongerEnabled(string $promotio
      */
     public function iShouldSeeExactlyNotifications(int $count): void
     {
-         assert(count($this->session->getPage()->findAll('css', '.sylius-flash-message')) === $count);
+        Assert::eq(count($this->session->getPage()->findAll('css', '.sylius-flash-message')), $count);
     }
 
     /**
      * @Then I should not see any notifications
      */
     public function iShouldNotSeeAnyNotifications(): void
     {
-        assert(count($this->session->getPage()->findAll('css', '.sylius-flash-message')) === 0);
+        Assert::eq(count($this->session->getPage()->findAll('css', '.sylius-flash-message')), 0);
```

Comment:
And you can merge this method with the one above by setting the default value `int $count = 0`

Reply:
Yeah, we should have something like `countFlashMessages` in the page class 👌